addons/hr_timesheet/views/hr_timesheet_portal_templates.xml

```diff
@@ -95,16 +95,6 @@
                                     </th>
                                 </t>
                             </tr>
-                            <tr t-else="">
-                                <div style="text-align: right;" class="mr-2 mb-1 text-muted">
-                                    <t t-if="is_uom_day">
-                                        Total: <span t-esc="timesheets._convert_hours_to_days(hours_spent)" t-options='{"widget": "timesheet_uom"}'/>
-                                    </t>
-                                    <t t-else="">
-                                        Total: <span t-esc="hours_spent" t-options='{"widget": "float_time"}'/>
-                                    </t>
-                                </div>
-                            </tr>
```

Comment:
?

Reply:
The previous code I removed. Now, I have added a new code instead of removing the else part of code.
Please check it.
Thanks